scalardb-test/src/main/java/kelpie/scalardb/Common.java

```diff
@@ -36,31 +36,26 @@ public static DistributedStorage getStorage(Config config) {
     return factory.getStorage();
   }
 
-  public static DistributedTransactionManager getTransactionManager(
-      Config config, String keyspace, String table) {
+  public static DistributedTransactionManager getTransactionManager(Config config) {
     DatabaseConfig dbConfig = getDatabaseConfig(config);
     TransactionFactory factory = new TransactionFactory(dbConfig);
-    DistributedTransactionManager manager = factory.getTransactionManager();
-    manager.with(keyspace, table);
-    return manager;
+    return factory.getTransactionManager();
   }
 
   public static TwoPhaseCommitTransactionManager getTwoPhaseCommitTransactionManager1(
-      Config config, String keyspace, String table) {
-    return getTwoPhaseCommitTransactionManager(getDatabaseConfig1(config), keyspace, table);
+      Config config) {
+    return getTwoPhaseCommitTransactionManager(getDatabaseConfig1(config));
   }
 
   public static TwoPhaseCommitTransactionManager getTwoPhaseCommitTransactionManager2(
-      Config config, String keyspace, String table) {
-    return getTwoPhaseCommitTransactionManager(getDatabaseConfig2(config), keyspace, table);
+      Config config) {
+    return getTwoPhaseCommitTransactionManager(getDatabaseConfig2(config));
   }
 
   private static TwoPhaseCommitTransactionManager getTwoPhaseCommitTransactionManager(
-      Properties properties, String keyspace, String table) {
+      Properties properties) {
     TransactionFactory factory = TransactionFactory.create(properties);
-    TwoPhaseCommitTransactionManager manager = factory.getTwoPhaseCommitTransactionManager();
-    manager.with(keyspace, table);
```

Comment:
Stopped using the `TwoPhaseCommitTransactionManager.with()` method since it's deprecated.